projects/pastanaga/src/lib/button/button.component.scss

```diff
@@ -134,6 +124,15 @@ pa-button {
             color: $red-dark;
         }
     }
+    &#{&}-contrast {
+        color: $white;
```

Comment:
should be `$button-contrast-color` (same for colors below, use the proper variables)